.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2.1
+
+orbs:
+  python: circleci/python@1.5.0
+  browser-tools: circleci/browser-tools@1.4.0
+
+jobs:
+  test-and-lint:
+    docker:
+      - image: cimg/python:3.9
+    steps:
+      - browser-tools/install-chrome
+      - browser-tools/install-chromedriver
+      - checkout
+      - python/install-packages:
+          pkg-manager: pip
+          pip-dependency-file: requirements-dev.txt
+      - run:
+          name: Lint with flake8
+          command: |
+            flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+            flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
```

Comment:
Try to make any commands we're going to run on CI into `package.json` scripts, so it's easy for users to replicate them locally. Even for something as simple as the `pytest` command below - which you wouldn't want to run exactly as is locally since you don't want `--headless` - it's nice to have a package script like `"test": "pytest"` then call it in CI like `npm run test -- --headless`

Reply:
Thanks for the suggestion! just changed